### PR TITLE
Added blooddrinker and updated spell statistic icons

### PR DESCRIPTION
**Pull request by yajinni**

### CODE REVIEW

File: src/Parser/BloodDeathKnight/CombatLogParser.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ class CombatLogParser extends CoreCombatLogParser {
     OssuaryUptime: OssuaryUptime,
     WastedDeathAndDecay: WastedDeathAndDecay,
     RunicPowerWasted: RPWasted,
+    BlooddrinkerTicks: BlooddrinkerTicks,
```
Comment (rp4rk) [inline src/Parser/BloodDeathKnight/CombatLogParser.js]: These should be named like:
`blooddrinkerTicks: BlooddrinkerTicks,`

This is a pretty ubiquitous naming convention, the capital at the start means it refers to the class itself, whereas the lack indicates that it is an instance of that class.

File: src/Parser/BloodDeathKnight/Modules/Features/BloodPlagueUptime.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ class BloodPlagueUptime extends Module {
         icon={<SpellIcon id={SPELLS.BLOOD_PLAGUE.id} />}
         value={`${formatPercentage(bloodplagueUptime)} %`}
         label='Blood Plague uptime'
+        tooltip={`Provides small amount of damage and healing. Auto attacks against an infected target can proc Crimson Scorge.`}
```
Comment (rp4rk) [inline src/Parser/BloodDeathKnight/Modules/Features/BloodPlagueUptime.js]: Crimson Scourge? 😄

File: src/Parser/BloodDeathKnight/Modules/Features/BlooddrinkerTicks.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    abilityTracker: AbilityTracker,
+    combatants: Combatants,
+  };
+
+  _totalTicks = 0;
+  _totalCasts=0;
+  _currentTicks = 0;
+  _wastedTicks = 0;
+  _ruinedCasts = 0;
+  BLOODDRINKER_TICKS_PER_CAST = 4 ;
```
Comment (rp4rk) [inline src/Parser/BloodDeathKnight/Modules/Features/BlooddrinkerTicks.js]: Move this out of the class and designate it as `const` or make it a static.

File: src/Parser/BloodDeathKnight/Modules/Features/BloodPlagueUptime.js (hunk 2 above)
Comment (yajinni) [reply src/Parser/BloodDeathKnight/Modules/Features/BloodPlagueUptime.js]: I wonder if i can add a spell checking to my IDE lol

**Comment by MartijnHols** [comment]
Thanks!

**Pull request merged**